file_store/src/file_sink.rs

```diff
@@ -16,6 +16,12 @@ use tokio::{
 use tokio_util::codec::{length_delimited::LengthDelimitedCodec, FramedWrite};
 
 pub const DEFAULT_SINK_ROLL_MINS: i64 = 3;
+
+#[cfg(not(test))]
+pub const DEFAULT_SINK_CHECK_MILLIS: i64 = 60_000;
```

Comment:
Rename without DEFAULT prefix since this is not looked up anywhere

Reply:
Fixed 536f3c1